src/HotChocolate/MongoDb/src/Data/ErrorHelper.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Linq;
+using HotChocolate.Data;
+using HotChocolate.Data.Filters;
+using HotChocolate.Data.Sorting;
+using HotChocolate.Language;
+using HotChocolate.Data.Mongodb;
+using HotChocolate.Types;
+
+namespace HotChocolate.Data.MongoDb
+{
+    internal static class ErrorHelper
+    {
+        public static IError CreateNonNullError<T>(
+            IFilterField field,
+            IValueNode value,
+            IFilterVisitorContext<T> context)
+        {
+            IFilterInputType filterType = context.Types.OfType<IFilterInputType>().First();
+
+            return ErrorBuilder.New()
+                .SetMessage(
```

Comment:
Every error going forward needs an error code ....

The error code needs to be added to our error code list in the root.